migration-guide/redhat-to-ibm.adoc

```diff
@@ -0,0 +1,426 @@
+By remaining a functionally equivalent product to the Red Hat offering, customers can easily migrate from the former Red Hat offering once they have purchased an IBM subscription for {PRODUCT}.
+This section provides guidance on how to migrate from the former Red Hat offerings of {RH_PAM} / {RH_DM} to {PRODUCT} v{VERSION_OLD}.
+It is recommended that you follow this migration path before attempting to migrate to the next generation of the product known as {PRODUCT} v{VERSION}.  
+
+== Prerequisites
+
+If your current application is based on the legacy product known as {RH_BPMS} you must first bring your installation up to the latest release of {RH_PAM} / {RH_DM} v7 before performing any migration to {PRODUCT}.
+Please refer to the existing https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.0/html/migrating_from_red_hat_jboss_bpm_suite_6.4_to_red_hat_process_automation_manager_7.0/index[Red Hat Documentation] in order to facilitate migrating from {RH_BPMS} to {RH_PAM}/{RH_DM} v7.x.
+From there you are free to migrate to any release of {PRODUCT}.
+
+The following tools are required in order to migrate to {PRODUCT_SHORT} v{VERSION}:
+
+include::../partials/prerequisites.adoc[]
+
+== Procedure
+
+The following steps are typically involved with migrating from {RH_PAM} / {RH_DM} to {PRODUCT_LONG}:
+
+=== Step 1: Download Product Archives
+
+The first step in migration is to to download the product archives from the either {PRODUCT_FIX_CENTRAL_PAGE}[IBM Fix Central] _(which requires an active support subscription to {PRODUCT})_ or via the {DEV_PROGRAM_PAGE}[IBM Business Automation Manager Open Editions Developer Program] _(subject to its terms)_.
+From there, depending on what you need to get, you will download a few files _(replace with most current version)_.
+The most common ones are highlighted below, but download what makes the most sense for your environment. 
+
+.{PRODUCT_SHORT} Product Archives
+image::bamoe-downloads.png[Product Archives]
+
+The following archives are tyipcally used in migrating to {PRODUCT_SHORT}:
+
+.Common Product Archives
+|===
+| Product Archive | Description
+
+| ILMT_for_IBAMOE_**{VERSION**}.zip   
+| This archive is used by IBM ILMT in order to measure the usage deployed in an environment of the product
+
+| bamoe-**{VERSION**}-add-ons.zip   
+| This archive contains things like the Database Migration Scripts (found at path /bamoe-**VERSION**-add-ons/process-migration/upgrade-scripts/**Database**)
+
+| bamoe-**{VERSION**}-business-central-eap7-deployable.zip   
+| This archive is required if you are running Business Central on your JBoss Enterprise Application Platform (EAP)
+
+| bamoe-**{VERSION**}-kie-server-ee8.zip   
+| This archive is required if you are running running KIE Server on JBoss EAP
+
+| bamoe-**{VERSION**}-maven-repository.zip   
+| This archive is required if you need to have a disconnected loading of the Maven repository in your environment
```

Comment:
this is not disconnected; this is just local maven artificats for the binaries Red Hat builds. for disconnected there's a offliner tool

Reply:
@blulas this needs to be addressed.